flink-clients/src/main/java/org/apache/flink/client/deployment/application/ApplicationDispatcherBootstrap.java

```diff
@@ -130,12 +132,17 @@ public void stop() {
 
 		return applicationCompletionFuture
 				.handle((r, t) -> {
+					final ApplicationStatus applicationStatus;
 					if (t != null) {
-						LOG.warn("Application FAILED: ", t);
+						applicationStatus = t instanceof CancellationException
```

Comment:
I have tested on a YARN cluster, it could not take effect here. When i cancel the Flink job, the application status is still `FAILED`.

Reply:
Thanks for testing it @wangyang0918 . Now it is fixed :)